packages/codemods/src/cli.ts

```diff
@@ -0,0 +1,56 @@
+import { $ } from 'execa';
+import sade from 'sade';
+import { fs, path } from 'zx';
+import { _dirname, errorMessage } from './helpers';
+
+const packageJson = fs.readJsonSync(path.resolve(_dirname, './package.json'));
+
+export default async function cli(args: string[]) {
+  sade('spirit-codemods', true)
+    .version(packageJson.version)
+    .describe(packageJson.description)
+    .option('-p, --path', 'Path to the code to be transformed')
+    .example('-p ./')
+    .option('-c, --codemod', 'Codemod name to run')
+    .example('-c v2/react/codemodName')
+    .option('-e, --extensions', 'Extensions to look for when transforming files, default: ts,tsx,js,jsx')
+    .example('-e ts, tsx, js, jsx')
+    .option('--ignore-pattern', 'Ignore files or directories, default: **/node_modules/**')
+    .example('-i **/node_modules/**')
+    .option('--parser', 'Parser to use (babel, ts, tsx, flow), default: babel')
+    .example('--parser ts')
+    .action(async ({ path: codePath, codemod, extensions, ignorePattern, parser }) => {
+      const defaultExtensions = 'ts,tsx,js,jsx';
+      const defaultIgnore = '**/node_modules/**';
+      const defaultParser = 'babel';
+
+      if (!codePath || !fs.existsSync(codePath)) {
+        errorMessage(codePath);
+        errorMessage('Please provide a valid path');
+        process.exit(1);
+      }
+
+      if (!codemod) {
+        errorMessage('Please provide a codemod name');
+        process.exit(1);
+      }
+
+      const codemodPath = path.resolve(_dirname, `./transforms/${codemod}.ts`);
+
+      if (!fs.existsSync(codemodPath)) {
+        errorMessage('Codemod does not exists');
+        process.exit(1);
+      }
+
+      const { stdout } = await $`jscodeshift --transform ${codemodPath} --extensions ${
+        extensions || defaultExtensions
+      } --ignore-pattern=${ignorePattern || defaultIgnore} --parser=${parser || defaultParser} ${codePath}`;
+
+      // stdout object from jscodeshift
+      // eslint-disable-next-line no-console
+      console.log(stdout);
```

Comment:
I suggest creating a `logMessage` or something similar the same way as we have the `infoMessage`. Thus we can ignore the console in one file only.

Reply:
`logMessage` added to message helpers and used there